receptor-python-worker/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+test_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Is this needed for Python 2 compat? If not — use `pathlib.Path` internally. Also, having `__init__.py` in the test dir changes how `pytest` treats things. It's usually discouraged. And common helpers and configurations would go into `conftest.py` instead.